test/plugin/scenarios/postgresql-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+    applications:
+    - {postgresql-scenario: nq 0}
+    instances:
+    - {postgresql-scenario: 1}
+    operationNames:
+    - postgresql-scenario: [/postgresql-scenario/case/healthcheck, /postgresql-scenario/case/postgres,
+                            PostgreSQL/JDBI/PreparedStatement/execute, PostgreSQL/JDBI/CallableStatement/execute,
+                            PostgreSQL/JDBI/Statement/execute, PostgreSQL/JDBI/Connection/close]
+    heartbeat: []
+segmentItems:
+- applicationCode: postgresql-scenario
+  segmentSize: ge 1
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /postgresql-scenario/case/healthcheck
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 1
+      componentName: ''
+      isError: false
+      spanType: Entry
+      peer: ''
+      peerId: 0
+      tags:
+      - {key: url, value: 'http://localhost:8080/postgresql-scenario/case/healthcheck'}
+      - {key: http.method, value: HEAD}
+  - segmentId: not null
+    spans:
+    -
+        operationName: PostgreSQL/JDBI/PreparedStatement/execute
```

Comment:
This structure seems not formatted well. Didn't expect a new line.

Reply:
ACK